lib/puppet/functions/default_content.rb

```diff
@@ -1,10 +1,11 @@
 Puppet::Functions.create_function(:default_content) do
-  # @summary Takes an optional content and an optional template name and
-  # returns the contents of a file.
-  # @param :content
-  # @param :template_name
+  # @summary Takes an optional content and an optional template name and returns the contents of a file.
+  #
+  # @param content
+  # @param template_name
   # @return [String]
```

Comment:
Below it's `Variant[Undef,String]` (i.e. `Optional[String]`). Should this be matched? It looks like it can be removed because puppet-strings autodetects it.

Reply:
I'd like to say I've 'fixed' this... but actually, I think all I've done is find another puppet-strings bug. :(